pkgs/by-name/u2/u2f-emulated/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  clangStdenv,
+  fetchFromGitHub,
+  openssl,
+}:
+
+clangStdenv.mkDerivation {
+  pname = "u2f-emulated";
+  version = "unstable-2020-06-12";
+
+  src = fetchFromGitHub {
+    owner = "Agnoctopus";
+    repo = "u2f-emulated";
+    rev = "b371044b7a1fa2bc0e4f2d0ee3a6b7482d24041d";
+    hash = "sha256-ndKxplsniO6ECePHp9/OnxFcYqRlO/poGpLR3pPahSk=";
+  };
+
+  patches = [
+    ./stdlib.patch
```

Comment:
The QEMU change seems fine, but what do we need this package for?  It's not explained, and it seems to have been abandoned upstream for 4.5 years.

Reply:
You're right, we don't need it for the qemu patch. I'll split it out to a new PR.